include/swift/AST/ASTContext.h

```diff
@@ -1000,7 +1000,7 @@ class ASTContext final {
   ///
   /// This is an implementation detail, access via
```

Comment:
This comment is not relevant anymore. Now this is the main API for getting the map.